content/realtime/channels.textile

```diff
@@ -236,6 +236,149 @@ channel.subscribe { message in
 }
 ```
 
+h4(#rewind). Subscribing to a channel with the rewind option enabled
+
+It is possible to subscribe to a channel and obtain messages from that channel's "history":/realtime/history with a single API request. To do so you would use the "rewind":/realtime/channel-params#rewind feature. This will result in the specified number of messages from "history":/realtime/history being returned as part of the subscription process.
+
+minimize. View details
+  "Rewind":/realtime/channel-params has two methods of obtaining messages from history. You can retreive a specified number of messages from history or messages from a set period of time into the past.
+
+  In Ably libraries older than @v1.2@, you can specify the "rewind":/realtime/channel-params parameter in the "metadata":/realtime/channel-metadata of the channel name, appending it at the end of any metadata otherwise specified. For example, if you have no metadata and wish to subscribe to channel @my_channel@ and fetch the most recent message from "history":/realtime/history, you would specify the channel as @[?rewind=1]my_channel@. If the channel had some metadata, @[some_metadata]my_channel@, you would apply "rewind":/realtime/channel-params with @[some_metadata?rewind=1]my_channel@. For more details, look at our "channel paramater":/realtime/channel-params documentation.
```

Comment:
I'm not really a fan of expressing this as "libraries older than 1.2". There isn't a 1.2 release anywhere, or even announced, yet we're referring to it here.

Personally I think here we should be saying that you can specify the rewind param using the channel params mechanism. In current Ably libraries, or when using the service without a library, this is by qualifying the channel name. There is a future library release planned in which it will be possible to specify channel params directly via the API.

Then, when we do release 1.2, we can update the text to say "In Ably library versions 1.2 or later, you can specify the rewind parameter as a channel param [ref]. In library versions earlier than 1.2, channel params are expressed by qualifying the channel name [ref] ....".

> "metadata":/realtime/channel-metadata of the channel name

This name qualifier is not the same as channel metadata. (Channel Metadata refers to the information such as activity and occupancy.) This text needs to use the correct term (refer back to that text) but it's going to be something like "you can specify the "rewind":/realtime/channel-params parameter by encoding it as a param in the qualifier part of the channel name ...".